test/state/state.cpp

```diff
@@ -75,11 +91,32 @@ evmc_message build_message(const Transaction& tx, int64_t execution_gas_limit) n
 /// Validates transaction and computes its execution gas limit (the amount of gas provided to EVM).
 /// @return  Execution gas limit or transaction validation error.
 std::variant<int64_t, std::error_code> validate_transaction(const Account& sender_acc,
-    const BlockInfo& block, const Transaction& tx, evmc_revision rev,
-    int64_t block_gas_left) noexcept
+    const BlockInfo& block, const Transaction& tx, evmc_revision rev, int64_t block_gas_left,
+    int64_t blob_gas_left) noexcept
 {
+    const auto blob_gas_price = fake_exponential(1, block.excess_blob_gas, 3338477);
```

Comment:
let's define these 2 constants somewhere

Reply:
Do you think we should go with Python naming convention for constants to be consistent with the spec? (sent from CLion PR review)

Reply:
BTW. We should have some common place to define constants like those two.